src/BuildingRegistry/Building/BuildingUnit.cs

```diff
@@ -416,6 +416,16 @@ public void DetachAddressBecauseAddressWasRemoved(AddressPersistentLocalId addre
             Apply(new BuildingUnitAddressWasDetachedBecauseAddressWasRemoved(_buildingPersistentLocalId, BuildingUnitPersistentLocalId, addressPersistentLocalId));
         }
 
+        public void ReplaceAddressAttachmentFromBuildingUnitBecauseStreetNameWasReaddressed(AddressPersistentLocalId sourceAddressPersistentLocalId, AddressPersistentLocalId destinationAddressPersistentLocalId)
```

Comment:
Can we rename sourceAddressPersistentLocalId and destinationAddressPersistentLocalId to new and previous?